pkg/daemon/ceph/model/pool.go

```diff
@@ -12,17 +12,20 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+
+Some of the code below came from https://github.com/digitalocean/ceph_exporter
```

Comment:
what in this file came from ceph_exporter?

Reply:
hmm, copy/paste issue. I need to remove all the ceph_exporter license comments with the removal of the collector